File: backend/src/main/java/com/coing/domain/coin/candle/dto/CandleDto.java
package com.coing.domain.coin.candle.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CandleDto {
	@JsonProperty("market")
	private String code;

	@JsonProperty("candle_date_time_utc")
	private String candleDateTimeUtc;

	@JsonProperty("opening_price")
	private double open;

	@JsonProperty("high_price")
	private double high;

	@JsonProperty("low_price")
	private double low;

	@JsonProperty("trade_price")
	private double close;

	@JsonProperty("candle_acc_trade_volume")
	private double volume;

	@JsonProperty("timestamp")
	private long timestamp;
}

File: backend/src/main/java/com/coing/domain/coin/candle/dto/CandleDto.kt
package com.coing.domain.coin.candle.dto

import com.coing.global.annotation.NoArg
import com.fasterxml.jackson.annotation.JsonProperty

@NoArg
data class CandleDto(
	@JsonProperty("market")
	var code: String? = null,

	@JsonProperty("candle_date_time_utc")
	var candleDateTimeUtc: String? = null,

	@JsonProperty("opening_price")
	var open: Double = 0.0,

	@JsonProperty("high_price")
	var high: Double = 0.0,

	@JsonProperty("low_price")
	var low: Double = 0.0,

	@JsonProperty("trade_price")
	var close: Double = 0.0,

	@JsonProperty("candle_acc_trade_volume")
	var volume: Double = 0.0,

	@JsonProperty("timestamp")
	var timestamp: Long = 0L
)
